lab_1_classify_by_unigrams/main.py

```diff
@@ -135,6 +233,29 @@ def preprocess_profile(profile: dict) -> dict[str, str | dict] | None:
     In case of corrupt input arguments or lack of keys 'name', 'n_words' and
     'freq' in arguments, None is returned
     """
+    if not isinstance(profile, dict):
+        return None
+    if (len(profile.keys()) != 3
+            or not all(x in profile for x in ['freq', 'name', 'n_words'])):
+        return None
+    processed_profile = {'name': profile['name'], 'freq': {}}
+    dictionary = {}
+    for unigram in profile['freq'].keys():
+        if unigram.isalpha():
+            pass
```

Comment:
why pass?

Reply:
I wanted to fix it somehow, that suddenly some kind of non-letter would come across again, so that I could get rid of it... although it's true, it's a bit pointless here...

done